Makefile

```diff
@@ -685,6 +685,16 @@ CPPINCLUDES =
 FCINCLUDES =
 LIBS =
 
+# Optional MUSICA support for chemistry
+ifeq "$(shell echo $(MUSICA) | tr '[:upper:]' '[:lower:]')" "true"
+ifeq ($(shell pkg-config --exists musica-fortran && echo yes || echo no), no)
+$(error "musica-fortran package is not installed. Please install it to proceed.")
+endif
+	MUSICA_FCINCLUDES += $(shell pkg-config --cflags musica-fortran)
+	MUSICA_LIBS += $(shell pkg-config --libs musica-fortran)
+	MUSICA_FFLAGS = -DMPAS_USE_MUSICA
+endif
+
```

Comment:
It could help to keep the MUSICA changes more consolidated if we moved this logic around line 885 (new line number) where we have the logic to append to `FCINCLUDES`, `LIBS`, and `FFLAGS`.

Reply:
so moved